news/1130.feature

```diff
@@ -0,0 +1 @@
+This change will allow disabling the configuration of Python's logging subsystem.
```

Comment:
Maybe something like the following? (to be more consistent with the other news entries, also no comma at the end.)

It is now possible to disable Hydra's logging configuration